app/src/main/java/com/willowtree/vocable/presets/NumberPadFragment.kt

```diff
@@ -0,0 +1,74 @@
+package com.willowtree.vocable.presets
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.GridLayout
+import androidx.core.os.bundleOf
+import androidx.core.view.updateMargins
+import com.willowtree.vocable.BaseFragment
+import com.willowtree.vocable.R
+import com.willowtree.vocable.customviews.VocableButton
+import com.willowtree.vocable.databinding.FragmentNumberPadBinding
+import com.willowtree.vocable.databinding.PhraseButtonBinding
+import com.willowtree.vocable.room.Phrase
+
+class NumberPadFragment: BaseFragment() {
+
+    companion object {
+        private const val KEY_PHRASES = "KEY_PHRASES"
+        const val MAX_PHRASES = 12
+
+        fun newInstance(phrases: List<Phrase>): NumberPadFragment {
+            return NumberPadFragment().apply {
```

Comment:
Since this is single expression, you can just do
```kotlin
fun newInstance(phrases: List<Phrase>) = NumberPadFragment().apply {
                arguments = bundleOf(KEY_PHRASES to ArrayList(phrases))
            }
```